### PR TITLE
Fixed #25655: Dropped support for GEOS < 3.3.

### PR DESCRIPTION
**Pull request by sir-sigurd**

### CODE REVIEW

File: django/contrib/gis/geos/prototypes/prepared.py [hunk 1]
```diff
@@ -22,7 +22,6 @@ class PreparedPredicate(GEOSFuncFactory):
 prepared_covers = PreparedPredicate('GEOSPreparedCovers')
 prepared_intersects = PreparedPredicate('GEOSPreparedIntersects')
```
Comment (claudep) [inline django/contrib/gis/geos/prototypes/prepared.py]: Let's delete that empty line and reorder the lines alphabetically.

File: docs/ref/contrib/gis/install/geolibs.txt [hunk 2]
```diff
@@ -8,7 +8,7 @@ geospatial libraries:
 ========================  ====================================  ================================  ============================
 Program                   Description                           Required                          Supported Versions
 ========================  ====================================  ================================  ============================
-:doc:`GEOS <../geos>`     Geometry Engine Open Source           Yes                               3.4, 3.3, 3.2
+:doc:`GEOS <../geos>`     Geometry Engine Open Source           Yes                               3.4, 3.3
```
Comment (claudep) [inline docs/ref/contrib/gis/install/geolibs.txt]: Not to do in this patch, but if anyone could test with GEOS 3.5, would be nice to add it here.

**Comment by claudep** [comment]
Apart the minor issue with `prepared.py`, the patch is ready.

File: docs/ref/contrib/gis/install/geolibs.txt (hunk 2 above)
Comment (sir-sigurd) [inline docs/ref/contrib/gis/install/geolibs.txt]: I'm using GEOS 3.5 and tests are passed.

Comment (claudep) [inline docs/ref/contrib/gis/install/geolibs.txt]: Great. If you create another PR, I will commit it (no need for a ticket, just ping me on the PR).

**Comment by claudep** [comment]
Oh, and just a small nit-pick: could you separate the ticket number from the message by `--` instead of `:`? (`Fixed #25655 -- ...`).

**Comment by claudep** [comment]
Merged in b78226cd3d, thanks!

**Pull request closed without merging**